fdmprinting/archive_retrieve.py

```diff
@@ -179,9 +190,17 @@ async def send_printer_status(message):
                 title="🖨 " + printer + ": " + status["state"].title(),
                 color=0xFF5733
             )
-
-            value = ("""`{0}`\n{1} Min Remain\n\n_{2}_""".
+            
+            failure_code = check_for_fail_reason(status)
+            
+            if failure_code != 0:
+                
+                value = ("""`{0}`\n\n**{1}**\n\n_{3}_""".
+                     format(status["job"], get_bambu_error_msg(failure_code),status["mins"], status["dateLocal"]))
```

Comment:
`status["mins"]` is unused, so remove it (but per above, I think we should both refactor this to DRY it out and to possibly include `Min Remain` even when we have an error